### ISSUE TITLE
"Documents" section doesn't list any document in Toulouse env

### ISSUE THREAD

**Issue by MiskoG** [arkhn-oss/Cohort360#113]
Spotted by @elsiehoffet-94 
![image](https://user-images.githubusercontent.com/7942024/124491032-19ffa180-ddb3-11eb-9da8-ecaabbab8229.png)

Seems like we have a bad request on the _DocumentReference_ FHIR endpoint.
I think there is an easy fix but I'm not sure, what do you think @simonvadee ?

**Comment by simonvadee** [comment]
I identified the issue, it's partly due to a mapping issue on DocumentReference and also because the cohort app is not flexible at all. I'm fixing this thanks to @Peltier10 help.

**Comment by MiskoG** [comment]
Ok great ! Does fixing this imply only modifying the mapping, or also make Cohort more flexible in this case ?

**Comment by simonvadee** [comment]
We need to modify both. Should be done tomorrow.

**Comment by simonvadee** [comment]
don't have time to finish this sorry: For @Jasopaum :
in `src/components/Cohort/Documents/DocumentList/DocumentList.tsx`: the file url needs to start with `file://` and end with` .pdf`. We only use the first extension but sometimes there are multiple ones (for tls, there are three sections).
You need to adapt `src/components/Cohort/Documents/PdfRenderer/PdfRenderer.tsx` to accept multiple sections

**Comment by Jasopaum** [comment]
> don't have time to finish this sorry: For @Jasopaum :
> in `src/components/Cohort/Documents/DocumentList/DocumentList.tsx`: the file url needs to start with `file://` and end with` .pdf`. We only use the first extension but sometimes there are multiple ones (for tls, there are three sections).
> You need to adapt `src/components/Cohort/Documents/PdfRenderer/PdfRenderer.tsx` to accept multiple sections

I don't think this was really the answer to this issue. The problem was that the api response was an empty bundle when trying to sort on the search param date. I removed the sorting for now but we need to investigate

**Comment by MiskoG** [comment]
> I removed the sorting for now but we need to investigate

@Jasopaum Can we consider this initial bug/issue as fixed ? Is there any PR associated with the sort removing ?
